backend/src/main/java/com/chinguv45tier3team53/tripplannerrest/entity/Trip.java

```diff
@@ -0,0 +1,29 @@
+package com.chinguv45tier3team53.tripplannerrest.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Entity
+@Builder
```

Comment:
do you use the builder somewhere?

Reply:
Yes, for seeding the db, since we are still in the process of developing, I thought this would help, will remove unused imports at a later stage. 😄